File: src/ts/classes.ts
import * as process from "process";

class MontyPython{
    constructor(public FirstName:string, public LastName:string){
    }

    fullName(){
        return `${this.FirstName} ${this.LastName}`;
    }
}

let montyPython = new MontyPython("John", "Cleese");
process.stdout.write(`Monty Python is ${montyPython.fullName()}.\n`);

montyPython.FirstName = "Eric";
montyPython.LastName = "Idle";
process.stdout.write(`Now, Monty Python is ${montyPython.fullName()}.`);

File: src/py/classes.py
class MontyPython:
    def __init__(self, firstName, lastName):
        self.FirstName = firstName
        self.LastName = lastName
    
    def FullName(self):
        return self.FirstName + " " + self.LastName

montyPython = MontyPython("John", "Cleese")
print("Monty Python is", montyPython.FullName(), ".")

montyPython.FirstName = "Eric"
montyPython.LastName = "Idle"
print("Now, Monty Python is", montyPython.FullName(), ".")